unstract/flags/src/unstract/flags/clients/flipt_client.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import grpc
+
+from ..generated import flipt_pb2, flipt_pb2_grpc
+from .base_client import BaseClient
+
+
+class FliptClient(BaseClient):
```

Comment:
@athul-rs Good to add a doc string mentioning the purpose and/or how this client is different from others.

Reply:
updated

Reply:
updated